archey/output.py

```diff
@@ -65,17 +79,75 @@ def append(self, key, value):
     def output(self):
         """
         Finally render the output entries.
-        It handles text centering additionally to value and colors replacing.
+        First we get entries to add their outputs to the results.
+        It then handles text centering additionally to value and colors replacing.
         """
-        # Let's center the entries according to the logo (handles odd numbers)
-        self._results[0:0] = [''] * ((18 - len(self._results)) // 2)
-        self._results.extend([''] * (18 - len(self._results)))
```

Comment:
I've allowed myself to clean up/simplify your changes here a bit.
The behavior is **not** the same as yours. I'm really looking forward to hearing from you on this. Maybe corresponding test cases need to be updated too :roll_eyes:

Reply:
Yeah - I tried using TextWrapper but wasn't massively keen on how it wraps entire words, since it makes the IP address entry look a bit strange with a large gap truncated - it does look way better with other entries (like the GPU entry), though. Since the whole-word truncation only really affects how the IP address entry looks, we should probably stick with TextWrapper since it does make the other entries look better when truncated 👍